exonum/src/api/public/blockchain_explorer.rs

```diff
@@ -141,9 +142,13 @@ impl ExplorerApi {
             "Expected tx in database",
         );
 
-        Ok(self.blockchain.tx_from_raw(raw_tx.clone()).ok_or_else(|| {
-            ApiError::InternalError(format!("Service not found for tx: {:?}", raw_tx).into())
-        })?)
+        Ok(self.blockchain.tx_from_raw(raw_tx.clone()).or_else(
+            |error| {
+                Err(ApiError::InternalError(
+                    format!("{}, tx: {:?}", error.description(), raw_tx).into(),
```

Comment:
Hmmm... I don't think we should print `raw_tx`. `Debug` is auto-derived and it will be something like `RawMessage(sync::Arc<MessageBuffer { raw: [1, 123, 123, 321, 324, ....] }>)`